db/access.php

```diff
@@ -40,6 +41,7 @@
         'contextlevel' => CONTEXT_COURSE,
         'archetypes' => array(
             'editingteacher' => CAP_ALLOW,
+            'student' => CAP_ALLOW,
```

Comment:
@xpazv this comment seems to have been lost on that hanging commit:

> Student's are never allowed to access any of our products. Does that change this?

Reply:
Wi will set this capability to prohibit for student role

Reply:
Does that mean that this line needs to change?

Reply:
Yes, it will be changed in the next commit from our side.